kubernetes/samples/charts/apache/default-sample.md

```diff
@@ -0,0 +1,38 @@
+# Apache Load Balancer default sample
+## Configure Apache Webtier as Load Balancer for WLS Domains
+In this section we will demonstrate how to use Apache webtier to handle traffic to backend WLS domains.
+
+### 1. Install WebLogic Server Domain
+Now we need to prepare some backends for Apache to do load balancing.
+
+Create a WebLogic Server domain:
+- Under default namespace.
+- Domain name is 'domain1'.
+- Deploy a webapp  with url context 'testwebapp'.
+
+### 2. Pull Apache Webtier Docker Image
+Run the following commands to pull Apache webtier docker image from repositry manually.
```

Comment:
This is an internal repo. We need to at least mention that so that users from outside of Oracle is aware of this.

Reply:
When we have this sample in master branch users will not have access to the image in this repository.
Until we get approval to have an Apache image in the Docker Hub you should point to the GitHub project https://github.com/oracle/docker-images/tree/master/OracleWebLogic/samples/12213-webtier-apache for them to build their own Apache image.